packages/components/src/modal/README.md

```diff
@@ -187,10 +187,14 @@ Titles are required for accessibility reasons, see `aria.labelledby` and `title`
 
 -   Required: No
 
-#### `focusOnMount`: `boolean | 'firstElement'`
+#### `focusOnMount`: `boolean | 'firstElement'` | 'firstContentElement'
 
 If this property is true, it will focus the first tabbable element rendered in the modal.
 
+If set to `firstElement` it will focus the first focusable element anywhere within the Modal.
+
+If set to `firstContentElement` it will focus the first focusable element within the Modal's **content** (i.e. children).
```

Comment:
Might be worth adding a short sentence here explaining that it's the responsibility of the consumer to add at least one focusable element in `children` or the focus will be lost.

Reply:
Also, is "tabbable" not the more accurate term instead of "focusable"? If not, maybe the preceding sentence should be edited for consistency.

Reply:
✅ Reworded and edited for consistency.